packages/appium/lib/cli/parser.js

```diff
@@ -146,7 +146,7 @@ class ArgParser {
     return _.reduce(
       args,
       (unpacked, value, key) => {
-        if (hasArgSpec(key)) {
+        if (!_.isUndefined(value) && hasArgSpec(key)) {
```

Comment:
is it possible to set `null` as an arg value? I assume the check should look like `!_.isNil(value)`

Reply:
no.